packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -237,7 +239,47 @@ export const Product = ({
       productId
```

Comment:
This util won't serve this purpose correctly in the situation that the user holds an IAP subscription to a different product on the same productSet, which should also be blocked.